pkgs/servers/resgate/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
```

Comment:
It does not seem to be used, did you want to add some sort of `testVersion` as this is a Go binary?

Reply:
Copy-paste error 😞